### PR TITLE
[TIC-312] Add new arguments to `create_org` 

### PR DESCRIPTION
**Pull request by itailevi98**
https://propelauth.atlassian.net/browse/TIC-312

### CODE REVIEW

File: propelauth_py/api/org.py [hunk 1]
```diff
@@ -65,11 +65,22 @@ def _fetch_org_by_query(
 ####################
 #       POST       #
 ####################
-def _create_org(auth_url, integration_api_key, name, max_users=None):
-    url = auth_url + f"{ENDPOINT_PATH}/"
-    json = {"name": name}
-    if max_users is not None:
-        json["max_users"] = max_users
```
Comment (itailevi98) [inline propelauth_py/api/org.py:72]: Note: `max_users` did not seem to exist anymore in the request in the backend, hence it was removed.

Comment (andrew-propelauth) [reply propelauth_py/api/org.py:72]: We should actually just address this by adding max_users to the backend call. I pushed up a PR for it, and we can hold this one back until after we deploy.

**Review by andrew-propelauth — CHANGES_REQUESTED** [review]

File: propelauth_py/api/org.py (hunk 1 above)
Comment (itailevi98) [reply propelauth_py/api/org.py]: No problem, I've added it back.

**Review by andrew-propelauth — APPROVED** [review]
lgtm!

**Pull request merged**